cmds/uinit/uinit.go

```diff
@@ -28,7 +28,8 @@ const (
 )
 
 var (
-	startupCmds   = []string{"sos", "wifi", "upspin"}
+	userenv       = "user"
```

Comment:
It's actually more common to spell these as
userEnv
startupCmds

since it shows the break between different parts of the variable name into words. For sure you should have a change to startupcmds in a different PR, but I'd much prefer it if you made it userEnv.

Reply:
so take out the change to startupcmds and maybe spell it userEnv